src/frontend/src/components/FillForms/UserDataForm.tsx

```diff
@@ -67,15 +105,23 @@ class UserDataForm extends React.Component<Props, State> {
 
   public handleSubmit = (e: React.BaseSyntheticEvent) => {
     e.preventDefault();
-    return this.props.downloadExpungementPacket(
-      this.state.name,
-      this.state.dob,
-      this.state.mailingAddress,
-      this.state.phoneNumber,
-      this.state.city,
-      this.state.state,
-      this.state.zipCode
-    );
+    this.validateForm().then(() => {
```

Comment:
You don't need to `return this.validateForm()...`?

Reply:
I don't. The handler function doesn't need a return value.